### PR TITLE
AcmModal refactor

### PR DESCRIPTION
**Pull request by showeimer**
Follow pattern similar to AcmButton where we can re-use the types from Patternfly to pass through props and get full feature function from the PF components.

### CODE REVIEW

File: src/AcmModal/AcmModal.tsx [hunk 1]
```diff
@@ -1,45 +1,6 @@
...
-                <Button key="cancel" variant="link" onClick={props.cancel}>
-                    Cancel
-                </Button>,
-            ]}
-        >
-            {props.message}
-        </Modal>
-    )
+export function AcmModal(props: ModalProps) {
+    return <Modal {...props} ref={null} />
```
Comment (KevinFCormier) [inline src/AcmModal/AcmModal.tsx:5]: What is `ref={null}` for?

File: src/AcmModal/AcmModal.test.tsx [hunk 2]
```diff
@@ -3,36 +3,44 @@ import { render } from '@testing-library/react'
...
-        const { container } = render(
-            <AcmModal
-                open={false}
-                submit={() => null}
-                cancel={() => null}
-                title="Modal title"
-                message="Modal message"
-            />
-        )
+        const { container } = render(<Component />)
```
Comment (KevinFCormier) [inline src/AcmModal/AcmModal.test.tsx:39]: I'm confused as to how this is closed when `Component` has `isOpen={true}`.

Comment (showeimer) [reply src/AcmModal/AcmModal.test.tsx:39]: Good point; why are these tests passing 😂

File: src/AcmModal/AcmModal.tsx (hunk 1 above)
Comment (showeimer) [reply src/AcmModal/AcmModal.tsx:5]: Seems like it's not covered in the type; so I'm not sure what to do with it for now besides give it something that will make it happy.  Do you think we will need to support a ref on the modals?

**Review by jamestalton — APPROVED** [review]

**Pull request merged**